### ISSUE TITLE
Vetur has started replacing single quotes with double quotes in .vue files

### ISSUE THREAD

**Issue by jmcooper** [vuejs/vetur#763]
- [ x ] I have searched through existing issues
- [ x ] I have read through [docs](https://vuejs.github.io/vetur)
- [ x  ] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)

## Info

- Platform: Linux (Ubuntu 16.04)
- Vetur version: 0.11.7
- VS Code version: 1.22.2

## Problem

Vetur has started replacing all my single quotes with double quotes in my .vue files. I can even create a brand new project from scratch with just a single .vue file in the project (no eslint config or anything) and if I use single quotes inside the <script> tag in a .vue file it will get replaced with double quotes.  

## Reproducible Case
Install Vetur pluggin
Set `"editor.formatOnSave": true,`

I can create a new folder, add a single foo.vue file inside that folder and add this code:
```
<template>
</template>
<script>
let foo = 'bar';
</script>
```
And then the single quotes around 'bar' got replaced with double quotes.

If I clone https://github.com/octref/veturpack, it doesn't happen, but if I remove the .eslintrc and .prettierrc then it starts happening there also. It is almost as if it is finding a .eslintrc file somewhere, but I've looked up the path in parent folders and don't see one and I don't see one in my home folder (~/.eslintrc) either.

To verify that it was Vetur, I uninstalled all other extensions and when I disable Vetur the problem goes away.

**Comment by jmcooper** [comment]
It actually appears that the problem is bigger than just quotes. If I have Vetur installed and `"editor.formatOnSave": true` in my VS Code settings, it appears that a lot of things get formatted wrong even if I have an `.eslintrc` file.

After using the vue 3.0 cli to generate a project with linting I get an .eslintrc file like this:
```
{
  "root": true,
  "extends": [
    "plugin:vue/essential",
    "@vue/airbnb"
  ]
}```

If I have Vetur installed and formatOnSave set to true, it is removing dangling commas and changing single quotes to double quotes which is contrary to the airbnb settings.

**Comment by jmcooper** [comment]
Closed in favor of [#764](https://github.com/vuejs/vetur/issues/764)
